src/assets/CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Overview
+
+To convert svgs into usable react components, add the svgs to `src/assets` and run the script `npm run icons:format`. This will run through the svg files (in `src/assets`) and generate a corresponding jsx file (in `src/assets/formatted`) for any new svgs that were added. You will need to go into each new jsx file and make a few small edits to ensure everything works properly.
```

Comment:
can it generate tsx?

Reply:
It can, but I ran into some issues when I had it outputting `tsx` files. I don't think it will work for us out of the box using tsx files. However, svgr has the option to use your own custom template for generating these files, so that can be something I look into as a follow-on improvement